mne/channels/tests/test_standard_montage.py

```diff
@@ -172,15 +172,15 @@ def test_set_montage_artinis_basic():
 
     # Check a known location
     assert_array_almost_equal(raw.info['chs'][0]['loc'][:3],
-                              [0.0616, 0.075398, 0.07347])
+                              [0.054243, 0.081884, 0.054544])
```

Comment:
@HanBnrd @rob-luke are the `artinis` montages added in https://github.com/mne-tools/mne-python/pull/9141 in MNI / fsaverage space? If so, these new positions should be more accurate. Could you check if `plot_alignment` looks more accurate, or at least as accurate as before? The differences might be subtle (things will move slightly more anterior)...

Reply:
Hi @larsoner, yes [artinis-brite23.elc](https://github.com/mne-tools/mne-python/blob/main/mne/channels/data/montages/artinis-brite23.elc) and [artinis-octamon.elc](https://github.com/mne-tools/mne-python/blob/main/mne/channels/data/montages/artinis-octamon.elc) are using MNI coordinates. Sorry, what would you like to check exactly? Would you like us to run `plot_alignment` with fNIRS data for the Artinis montages and compare before and after your PR?

Reply:
> Would you like us to run plot_alignment with fNIRS data for the Artinis montages and compare before and after your PR?

Yeah that would be great. The changes suggest that they will shift anterior (and a little inferior) by ~1.5 cm but it should actually be more correct

Reply:
> It seems that after the PR, the sensors look like they're inside the head in the 3D vis

It could be that the head surface that the Artinis folks used to create the montage was slightly different. This seems to be the case for our EEG standard montages at least. It's tough to know which one is "more correct" I guess...